game/scripts/npc/items/item_radiance_3.txt

```diff
@@ -1,5 +1,5 @@
-"DOTAItems"
-{
+"DOTAItems"
+{

```

Comment:
No. They need to be removed.
`^M` means CR from CRLF line endings
but we want LF line endings